### PR TITLE
create: add --python option

### PR DESCRIPTION
**Pull request by dawidd6**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/master/Library/Homebrew/test/PATH_spec.rb).
- [x] Have you successfully run `brew style` with your changes locally?
- [x] Have you successfully run `brew tests` with your changes locally?

-----

Another PR that adds new option to `create` command. This time it's `--python`.

### CODE REVIEW

**Comment by jonchang** [comment]
Perhaps there could be an example resource block (commented-out?) that would help guide how `virtualenv_install_with_resources` can be used with Python dependencies?

**Comment by dawidd6** [comment]
@jonchang agreed, was thinking about it earlier. Change committed.

File: Library/Homebrew/formula_creator.rb [hunk 1]
```diff
@@ -134,7 +150,7 @@ def install
                                   "--prefix=\#{prefix}"
             # system "cmake", ".", *std_cmake_args
         <% end %>
-        <% if mode != :meson and mode != :go %>
+        <% if mode != :meson and mode != :go and mode != :python %>
```
Comment (MikeMcQuaid) [inline Library/Homebrew/formula_creator.rb]: Maybe just `if autotools || cmake` at this point?

Comment (dawidd6) [reply Library/Homebrew/formula_creator.rb]: Yes, this definitely looks better. Committed.

**Comment by MikeMcQuaid** [comment]
Great work again @dawidd6. I look forward to your next PRs 😄

**Comment by dawidd6** [comment]
Thanks @MikeMcQuaid! Next PR should arrive soon.

**Pull request merged**